core/domain/stats_jobs_one_off.py

```diff
@@ -214,6 +214,26 @@ def entity_classes_to_map_over(cls):
 
     @classmethod
     def prepare_map(cls, item):
+        """Returns a dict that represents the given model instance, so that it
```

Comment:
Doesn't seem to return a dict, based on the Returns docstring.

Reply:
Done